packages/katello/rubygem-pulp_docker_client/rubygem-pulp_docker.spec

```diff
@@ -0,0 +1,93 @@
+# Generated from pulp_docker_client-4.0.0b6.dev01562331743.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
Can you rename this script to follow the convention that the spec has the same name as the directory and package name?

Reply:
whooops! good catch